### DISCUSSION TITLE
Request support for multi-language function, thank you!

### DISCUSSION THREAD

**Discussion by wonly211** [valinet/ExplorerPatcher#359] [Ideas]
Request support for multi-language function, thank you!

**Comment by valinet** [comment]
To be honest, it won't be a priority at all; I don't have the manpower to take care of that as well. I also don't know many languages, things change pretty often, so it's very difficult to support this.

You can take the https://github.com/valinet/ExplorerPatcher/blob/master/ExplorerPatcher/settings.reg and translate it for personal use and that takes care of most* of the interface, for example. The rest of the string, not many, are scattered throughout the source code.

It's an educational hobby project...

**Comment by Galixte** [comment]
> Request support for multi-language function, thank you!

I have opened this topic to list all translations done: https://github.com/valinet/ExplorerPatcher/discussions/363. ;)
